wagtail/documents/static_src/wagtaildocs/scss/add-multiple.scss

```diff
@@ -1,4 +1,5 @@
-@import 'wagtailadmin/scss/helpers';
+@import '../../../../../client/scss/settings';
+@import '../../../../../client/scss/tools';
```

Comment:
Is there any way to avoid these changes to relative imports? Just makes this code much harder to refactor later.